nautobot_ssot/integrations/dna_center/constants.py

```diff
@@ -0,0 +1,33 @@
+"""Defines various constants used throughout the App."""
+
+from django.conf import settings
+
+# Import config vars from nautobot_config.py
+PLUGIN_CFG = settings.PLUGINS_CONFIG["nautobot_ssot"]
+
+DNAC_PLATFORM_MAPPER = {
```

Comment:
Can we use those from here? https://github.com/networktocode/netutils/pull/519

Reply:
Was waiting to have a new netutils version released as we'll need to pin the version in the dependencies.

Reply:
New release has been cut so I've updated the integration to use the mapping from there.